api/src/controllers/profile.ts

```diff
@@ -74,44 +73,41 @@ export const fetchProfileContacts = async (
 };
 
 export const updateProfileContacts = async (
```

Comment:
This fn might be getting a bit long. Consider breaking it down into smaller fn on your next tech debt cycle.

Reply:
100% agree, Next thing on my docket is extracting alot of the controller logic into separate components as it has definitely gotten quite bloated as the app has grown.